_layouts/post.html

```diff
@@ -11,7 +11,10 @@ <h1 class="post-title p-name" itemprop="name headline">{{ page.title | escape }}
         {{ page.date | date: date_format }}
       </time>
       {%- if page.author -%}
-        • <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name">{{ page.author | escape }}</span></span>
+        • {% for author in page.author %}
+          <span itemprop="author" itemscope itemtype="http://schema.org/Person">
+            <span class="p-author h-card" itemprop="name">{{ author }}</span></span>{% if forloop.last == false %}, {% endif %}
```

Comment:
You may use Liquid's WhitespaceControl feature to improve readability:
```suggestion
            <span class="p-author h-card" itemprop="name">{{ author }}</span></span>
            {%- if forloop.last == false %}, {% endif -%}
```